src/storage.ts

```diff
@@ -280,11 +291,18 @@ class AnnotationServerStorage {
                 Accept: "application/json",
                 "Content-Type": "application/json",
                 "X-CSRFToken": this.settings.csrf_token,
+                // ensure match with the previously fetched ETag
+                "If-Match": annotation.etag,
```

Comment:
repeated from `update` above, would it make sense to combine into a method to get the headers? (although not huge duplication)